packages/core-js/utils/events.js

```diff
@@ -9,6 +9,10 @@
 function findEvent({ receipt, eventName, contract = undefined }) {
   let eventsAreParsed = false;
   if (receipt.events !== undefined) {
+    const event = receipt.events.find((e) => e.event === eventName);
```

Comment:
If event is found, just return it

Reply:
Lol, this is all that was needed for the fix?